src/algorithms/ga.jl

```diff
@@ -24,19 +25,23 @@ function GA(
     M::MutationMethod
 )
     n = length(population)
-    fitnesses = []
-	for _ in 1:k_max
+
+    # NOTE: Is pop f32?
+    # TODO: create a timer macro
+    fitnesses = Vector{Float32}(undef, n)
```

Comment:
`population` is just a vector of vectors of any type, that's why we get it as `AbstractVector`. The `fitnesses` is a container for their evaluation, which normally would be Float (I assume it' s 64). I think we rather go with 64 for high precision applications?